examples/fcma/generate_fcma_data.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Generate simulated data for FCMA example
+
+Generate example data for FCMA analyses that uses fmrisim
+
+ Authors: Cameron Ellis (Princeton) 2017
+"""
+import logging
+import numpy as np
+from brainiak.utils import fmrisim as sim
+import nibabel
+import os
+
+logger = logging.getLogger(__name__)
+
+
+# Set the experimental parameters
+participants = 5
+epochs = 5
+dimensions = np.array([10, 10, 10])  # What is the size of the brain
+
+# Where will the data be stored?
+directory = 'examples/fcma/simulated/'
```

Comment:
This will not work as you expect if you execute the example from anywhere else but the root of the Git repository. You should test if `examples/fcma` exists. If not, exit and ask the user to pass the path as an argument; you can get it as `sys.argv[1]`.